### DISCUSSION TITLE
Allows to only show new version reminder for admins, but not for users

### DISCUSSION THREAD

**Discussion by hongbo-miao** [open-webui/open-webui#5869] [General]
**Is your feature request related to a problem? Please describe.**

Our Open WebUI shows new version reminders. I think it is very helpful for admins.
We got feedback from our end users that this reminder is very annoyed.

And Helm chart is not often up-to-date using latest version by default: https://github.com/open-webui/helm-charts
![image](https://github.com/user-attachments/assets/dd895530-1cdb-4312-9d6c-9fea90f3d932)
For example, v0.3.30 released 1 week ago, but Helm chart released today for this new version.

**Describe the solution you'd like**

I saw this ticket marked as not planned https://github.com/open-webui/open-webui/issues/5840
So could we only show new version reminder for admins, but not for end users? Thanks!

**Describe alternatives you've considered**

Before this new Helm chart release, to upgrade to latest version to remove the reminders, one workaround way is in Helm chart, I can set image tag version manually to `v0.3.30` to overwrite the old default tag version which was `v0.3.20`.

```yaml
image:
  repository: ghcr.io/open-webui/open-webui
  tag: v0.3.30
```

**Comment by tjbck** [comment]
https://github.com/open-webui/open-webui/blob/1d225dd804575af9ae5981528dfdce695f7f7040/src/routes/(app)/%2Blayout.svelte#L202

It's only shown to admins currently, unsure how your "users" are getting the notification here.

**Reply by hongbo-miao** [reply]
My bad, that is good to hear, thanks for the info @tjbck !
Confirmed the coworker who giving feedback is also admin. Confirmed the user role has no reminder.
